### PR TITLE
AO3-6432 Prevent orphan_account from being warned/banned/suspended

### PR DESCRIPTION
**Pull request by ellieyhcheng**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [x] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [x] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [x] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6432

## Purpose

Prevents `orphan_account` from being able to be warned/banned/suspended by an admin. Note that it also assumes that since `orphan_account` can never be banned or suspended, it can also never be unbanned or unsuspended.

## Testing Instructions

- Log in as a superadmin or policy_and_abuse admin
- Go to `orphan_account` page in admin view (endpoint `/admin/users/orphan_account`)
- Suspend or ban the account

It should give an error `orphan_account cannot be banned/suspended.`

## References

N/A

## Credit

ellieyhc (she/her)

### CODE REVIEW

**Comment by ellieyhcheng** [comment]
This only applies to "suspend", "ban", "spamban" actions. Admins can still write notes and give warnings to `orphan_account`. Should this ability also be disabled?

**Comment by sarken** [comment]
Thanks for the pull request!

I spoke to the Policy & Abuse chairs and they'd like to prevent warnings and notes as well, if you're willing to do that!

Also, just a heads up that we have `User.orphan_account` for identifying the orphan account, and you should use that wherever possible. You can find a few examples throughout the code.

**Comment by ellieyhcheng** [comment]
Modified the PR to prevent warnings and notes to `orphan_account` as well.

File: features/admins/users/admin_abuse_users.feature [hunk 1]
```diff
@@ -6,6 +6,7 @@ Feature: Admin Abuse actions
 
   Background:
     Given the user "mrparis" exists and is activated
+      And the user "orphan_account" exists and is activated
```
Comment (redsummernight) [inline features/admins/users/admin_abuse_users.feature:9]: Can you use the step "I have an orphan account" here instead?

**Review by redsummernight — COMMENTED** [review]
Looks good to me! Just a minor comment.

**Review by redsummernight — APPROVED** [review]
Thank you!

**Pull request merged**